lms/templates/courseware/courseware-chromeless.html

```diff
@@ -133,9 +133,27 @@
         var newWidth = contentElement.offsetWidth;
 
         if (eventType !== 'load' && newWidth === lastWidth && newHeight === lastHeight) {
+            // Check to see if the page is fully loaded
+            window.parent.postMessage("ready", document.referrer)
+
+            // Monitor when any achor tag is clicked, it is checked to make sure
+            // it is referencing an element's id (not an external website). If
+            // the href attribute is an id, the location of the selected focus
+            // element is sent through its offset attribute. The offset will
+            // allow the page to scroll to the location of the focus element so
+            // that it is at the top of the page.
+            
+            $('a').on("click", function(event){
+              if ($(this).attr('href')[0] === "#") {
```

Comment:
I'm curious how the href link starts with `#` - on the page mentioned in the ticket, it looks like `https://courses.edx.org/lots/of/stuff#actualanchor`?

Reply:
Thanks - makes sense!